Tailviewer.AcceptanceTests/BusinessLogic/LogFiles/TextLogFileTest.cs

```diff
@@ -65,8 +63,8 @@ public void TestCtor()
 			var info = new FileInfo(_fname);
 			_scheduler.RunOnce();
 
-			_file.GetValue(LogFileProperties.LastModified).Should().Be(info.LastWriteTime);
```

Comment:
This merge doesn't look right, GetValue(...) is the new way to retrieve data, please don't revert that.